src/go.mod

```diff
@@ -97,7 +97,6 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/denverdino/aliyungo v0.0.0-20191227032621-df38c6fa730c // indirect
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
-	github.com/distribution/reference v0.5.0 // indirect
```

Comment:
Why would this be removed? Is it not used indeed?

Reply:
We should not need this package, automatically remove after running`go mod tidy`.